pkg/asset/manifests/network.go

```diff
@@ -20,8 +21,8 @@ var (
 	noCfgFilename  = filepath.Join(manifestDir, "cluster-network-02-config.yml")
 	cnoCfgFilename = filepath.Join(manifestDir, "cluster-network-03-config.yml")
 	// Cluster Network MTU for AWS Local Zone deployments on edge machine pools.
-	ovnKNetworkMtuEdge   uint32 = 1200
-	ocpSDNNetworkMtuEdge uint32 = 1250
+	ovnKubernetesNetworkMtuEdge uint32 = 1200
+	openshiftSDNNetworkMtuEdge  uint32 = 1250
```

Comment:
@mtulio note that OpenShiftSDN support will be removed in 4.15, not sure if this still makes sense here [OCPSTRAT-959](https://issues.redhat.com/browse/OCPSTRAT-959)